guides/v2.2/b2b/negotiable-order-workflow.md

```diff
@@ -17,11 +17,11 @@ This topic describes how REST calls can be used to place items in a shopping car
 
 * You have [installed and enabled]({{ page.baseurl }}/comp-mgr/install-extensions/b2b-installation.html) {{site.data.var.b2b}}.
 * You have [created a company]({{ page.baseurl }}/b2b/company-object.html) and a [company user]({{ page.baseurl }}/b2b/company-object.html).
-* You have an integration or [admin authorization token]({{ page.baseurl }}/get-started/order-tutorial/order-admin-token.html) to make calls on behalf of seller, and a [customer token]({{ page.baseurl }}/get-started/order-tutorial/order-create-customer.html#get-token) to make calls on behalf of the company user.
+* You have an integration or [admin authorization token]({{ page.baseurl }}/rest/tutorials/orders/order-admin-token.html) to make calls on behalf of seller, and a [customer token]({{ page.baseurl }}/rest/tutorials/orders/order-create-customer.html#get-token) to make calls on behalf of the company user.
 
 ## Prepare the order
 
-The steps in this section are similar to those [Order Processing with REST APIs Tutorial]({{ page.baseurl }}/get-started/order-tutorial/order-intro.html), except that different products are added to the cart.
+The steps in this section are similar to those [Order Processing with REST APIs Tutorial]({{ page.baseurl }}/rest/tutorials/orders/order-intro.html), except that different products are added to the cart.
```

Comment:
Consider dropping "... with REST APIs" from the title

You also need a preposition + article before the reference:

> The steps in this section are similar to those **in the** Order Processing with REST APIs Tutorial...

Reply:
Added preposition/article.